tests/network-tests/src/scenarios/memberships.ts

```diff
@@ -24,4 +25,5 @@ scenario('Memberships', async ({ job }) => {
   job('transferring invites', transferringInvites)
   job('managing staking accounts', managingStakingAccounts)
   job('updating member verification status', updatingVerificationStatus).after(hireLeads)
+  job('updating member verify account status', updateValidatorAccount)
```

Comment:
```suggestion
  job('updating member validator account', updateValidatorAccount).after(hireLeads)
```
Also I think this job needs to be added the `full.ts` scenario too in order to be ran by the CI.